components/server/src/orgs/organization-service.ts

```diff
@@ -187,12 +188,31 @@ export class OrganizationService {
 
         try {
             await this.teamDB.transaction(async (db) => {
+                const members = await this.teamDB.findMembersByTeam(orgId);
+                const firstMember = members.filter((m) => m.userId !== BUILTIN_INSTLLATION_ADMIN_USER_ID).length === 0;
+                if (firstMember) {
+                    // first member (that is not an admin) is going to be an owner
+                    role = "owner";
+                    log.info({ userId: memberId }, "First member of organization, setting role to owner.");
+                }
```

Comment:
I'm seeing either:
 - the "name your org" screen
 - an error "can't establish a session" (or similar)

Papi logs say: "The user is not currently a member of this organization"


I think we're missing a `await this.teamDB.addMemberToTeam(memberId, orgId);` here.

Reply:
fixed